Apps/Gateway/src/ClientApp/src/stores/user.ts

```diff
@@ -0,0 +1,468 @@
+        oidcUserInfo.value = userInfo;
+        setLoadedStatus();
+    }
+
+    function setProfileUserData(userProfile: UserProfile) {
+        if (userProfile) {
+            PreferenceUtil.setDefaultValue(
+                userProfile.preferences,
+                UserPreferenceType.HideVaccineCardQuickLink,
+                "false"
+            );
+            PreferenceUtil.setDefaultValue(
+                userProfile.preferences,
+                UserPreferenceType.HideOrganDonorQuickLink,
+                "false"
+            );
+            PreferenceUtil.setDefaultValue(
+                userProfile.preferences,
+                UserPreferenceType.HideImmunizationRecordQuickLink,
+                "false"
+            );
+        }
+
+        user.value.acceptedTermsOfService = userProfile
+            ? userProfile.acceptedTermsOfService
+            : false;
+        user.value.hasTermsOfServiceUpdated = userProfile
+            ? userProfile.hasTermsOfServiceUpdated ?? false
+            : false;
+        user.value.lastLoginDateTimes = userProfile
+            ? userProfile.lastLoginDateTimes
+            : [];
+        user.value.closedDateTime = userProfile
+            ? userProfile.closedDateTime
+            : undefined;
+        user.value.preferences = userProfile ? userProfile.preferences : {};
+        user.value.hasEmail = !!userProfile?.email;
+        user.value.verifiedEmail = userProfile.isEmailVerified;
+        user.value.hasSMS = !!userProfile?.smsNumber;
+        user.value.verifiedSMS = userProfile.isSMSNumberVerified;
+        user.value.hasTourUpdated = userProfile.hasTourUpdated ?? false;
+
+        logger.verbose(`User: ${JSON.stringify(user.value)}`);
+
+        setLoadedStatus(patient.value.hdid !== undefined);
+    }
+
+    function setSmsResendDateTime(date: DateWrapper) {
+        smsResendDateTime.value = date;
+        setLoadedStatus();
+    }
+
+    function setUserError(errorMessage: string) {
+        error.value = true;
+        statusMessage.value = errorMessage;
+        status.value = LoadStatus.ERROR;
+    }
+
+    function setLoadedStatus(isCompletelyLoaded: boolean = true) {
+        error.value = false;
+        statusMessage.value = "Success";
+        status.value = isCompletelyLoaded
+            ? LoadStatus.LOADED
+            : LoadStatus.PARTIALLY_LOADED;
+    }
+
+    function bindUserPreference(userPreference: UserPreference) {
+        logger.debug(
+            `setUserPreference: preference.name: ${JSON.stringify(
+                userPreference.preference
+            )}, preference.value: ${JSON.stringify(userPreference.value)}`
+        );
+        user.value.preferences = Object.assign({}, user.value.preferences, {
+            [userPreference.preference]: userPreference,
+        });
+        setLoadedStatus();
+    }
+
+    function setPatient(incomingPatient: Patient) {
+        logger.debug(`setPatient: ${JSON.stringify(patient)}`);
```

Comment:
```suggestion
        logger.debug(`setPatient: ${JSON.stringify(incomingPatient)}`);
```

Assuming this is a typo resulting from the naming clash.